packages/Organisms/SiteHeader/index.vue

```diff
@@ -23,8 +23,15 @@
               <rpl-icon :symbol="menuButton[menuState].icon" color="white"></rpl-icon>
               <span>{{ menuButton[menuState].text }}</span>
             </button>
-            <!-- Logo -->
-            <div v-if="!menuContentOpen && logo" class="rpl-site-header__title">
+            <!-- Primary vic.gov.au Logo -->
+            <div v-if="!menuContentOpen" class="rpl-site-header__title rpl-site-header__logo-container--vic-logo-primary" :class = "{'rpl-site-header__logo-container--vic-logo-primary--cobrand' : (logo)}"> <!--If there is no cobrand logo, never hide the VIC logo-->
+              <rpl-link :href="vicLogoStatic.url">
+                <img :src="vicLogoStatic.location" :alt="vicLogoStatic.alt" />
+              </rpl-link>
+            </div>
```

Comment:
I think this change will add vic logo in front of all sites custom logo. So basically the result for all current sites will be:
- beta site will have two vic logos.
- Site like https://www.aboriginalheritagecouncil.vic.gov.au/ & https://www.policemuseum.vic.gov.au/ will have a vic logo even they don't have it now.

Reply:
I thought the displaying the vic logo at large viewports was part of the design guidelines?